src/Common/CmdletInfoHandler.cs

```diff
@@ -64,11 +64,11 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             }
             if (ClientRequestId != null)
             {
-                if (request.Headers.Contains("x-ms-client-request-id"))
+                if (request.Headers.Contains(ApiConstants.HeaderNameClientRequestId))
                 {
-                    request.Headers.Remove("x-ms-client-request-id");
+                    request.Headers.Remove(ApiConstants.HeaderNameClientRequestId);
                 }
-                request.Headers.TryAddWithoutValidation("x-ms-client-request-id", ClientRequestId);
+                request.Headers.TryAddWithoutValidation(ApiConstants.HeaderNameClientRequestId, ClientRequestId);
             }
```

Comment:
nit: It seems these code are similar in three places, how about move to helper method?

Reply:
It's current design problem. Ideally, this logic should only show in one place. but that needs cmdlet info handler and trancing inceptors to reference each other.  So, just copy code 3 times.

I don't hope to introduce additional helper class for this problem.

Reply:
Add extensions